src/Adapters/Laravel/PanManager.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pan\Adapters\Laravel;
+
+final class PanManager
+{
+    private int $max;
+
+    /**
+     * @var array<int, string>
+     */
+    private array $allowed;
+
+    private int $perMinute;
+
+    public function __construct()
+    {
+        /** @phpstan-ignore cast.int */
+        $this->max = (int) env('PAN_MAX', 50);
```

Comment:
`env` should really only be used inside configuration files[^1], consider creating a config file instead. 


[^1]: https://laravel.com/docs/11.x/configuration#configuration-caching